Interview/demo.md

```diff
@@ -22,62 +22,74 @@ Before proceeding to the section of your choice, please consider the following t
 
-* http://jsonplaceholder.typicode.com/posts
+* http://jsonplaceholder.typicode.com/photos
 
-When a post is tapped, you should go to the detail screen.
+When a photo is tapped, you should go to the Photo detail screen.
+Also, a photo that is marked favourite shows up first in the list followed by other photos.
 
-#### Detail screen
+#### Photo Detail screen
 
-A post detail screen will have:
+A Photo detail screen will have:
 
-* Its author.
-* Its description (via the `body`).
-* The number of comments it has.
+* The user the photo belongs to.
+    * Get the user id from the album(http://jsonplaceholder.typicode.com/albums/{:id})
+    * Get the User from the user detail (http://jsonplaceholder.typicode.com/user/{:id})
+* The title of the photo.
+* The number of comments on the photo:
+    * (https://jsonplaceholder.typicode.com/photos/{:id}/comments)
+* The ability to mark/unmark a photo as favourite.
 
-You can retrieve the remaining information from these API:
 
-* http://jsonplaceholder.typicode.com/users
-* http://jsonplaceholder.typicode.com/comments
 
 #### Requirements
 
 The following requirements should be met:
 
-* Use Swift 4.0 or above.
-* The information (posts and post details) should be available offline. It's assumed that, if it's the first time you are accessing the app and you are offline, you shouldn't see any data.
-* Add reloading mechanism e.g. button or `UIRefreshControl`. `Reachability` is not required.
+* Use Swift 5.0 or above.
+* No 3rd party libraries for networking
+* The information (photos and photo details) should be available offline. It's assumed that, if it's the first time you are accessing the app and you are offline, you shouldn't see any data.
+* A reloading mechanism e.g. button or `UIRefreshControl`. `Reachability` is not required.
 * **Have a point of synchronization (e.g. making two concurrent requests and waiting for both of them to finish).**
 * The code should be production grade.
 * It should compile and run.
+* Stutter free, crash free UI.
+* If you are applying for Senior, there are additional Requirements:
+    * You should use a reactive framework
+    * UI should be written in code
 
 ### 2. Previously written project
 
 We would be happy if you would submit a project you already have (for instance, a demo project for another company). Still, the project **must**:
 
-* Use Swift 4.0 or above.
+* Use Swift 5.0 or above.
+* No 3rd party libraries for networking
+* Should have clear instructions on how to run the project.
 * Have at least two distinct network calls.
 * Parse the network response and present the information to the user.
 * Have some sort of persistence mechanism.
 * Compile and run.
 * Have a point of synchronization (e.g. making two concurrent requests and waiting for both of them to finish).
+* If you are applying for Senior, there are additional Requirements:
+    * You should use a reactive framework
+    * UI should be written in code
```

Comment:
As I mentioned above. This is particularly harsh for a pre-existing project.

Reply:
Yeah fair enough 👍